i18n-helpers/src/lib.rs

```diff
@@ -160,6 +160,26 @@ pub enum Group<'a> {
     Skip(Vec<(usize, Event<'a>)>),
 }
 
+#[derive(Debug, Clone, PartialEq, Default)]
+pub struct Extra {
```

Comment:
Please add a doccomment here — the name `Extra` is quite generic, so it'll be hard for a casual reader to know what it's about :sweat_smile:

If the type is produced by a certain function, then it can be useful to link to that function — similarly, if it's supposed to be confused by just one function, then linking to it can be useful too.

Reply:
Hmm... now that I take a fresh look at this, I do not like the extra structure.  Let me absorb its functionality into Group::Translate.  I need to change Group::Translate from a tuple to a struct, but I think it's better than what I was doing before there.

Reply:
Alright, sounds good to me! Thanks for keeping working on this!